cmd/fever/cmds/run.go

```diff
@@ -1,9 +1,10 @@
 package cmd
 
 // DCSO FEVER
-// Copyright (c) 2017, 2018, 2019, DCSO GmbH
+// Copyright (c) 2017, 2018, 2019, 2020, DCSO GmbH
```

Comment:
Just a note that `Copyright (c) 2017, 2020, DCSO GmbH` should be enough. Having all years is going to be very long in 10 years ;)

Reply:
Got it.